www/src/js/templates/snaplist.hbs

```diff
@@ -1,23 +1,29 @@
-{{#if title}}
-<div class="b-headline">
-  <div class="b-layout__container">
-    <h1 class="b-headline__h1">{{ title }}</h1>
-    <div class="b-snaplist__actions">
-      {{#if canSort}}
-      <div class="b-button__group">
-        <div id="sortAlpha" class="b-button b-button_small{{#if isAlpha}} b-button_active{{/if}}">A-Z</div>
-        <div id="sortBytes" class="b-button b-button_small{{#unless isAlpha}} b-button_active{{/unless}}">kB</div>
+<div class="inner-wrapper">
+  {{#if title}}
+  <div class="twelve-col last-col">
```

Comment:
Good point. If you have a look at the fixup commit I've just pushed you'll see that I was able to add the grid classes to the `b-headline` element.

I tried doing the same with `b-snaplist` but any element which has a padding or margin set is going to conflict with the grid and I think we just have to suck it up in these situations and add more elements to deal with the grid layout.